src/lib/stores/cartStore.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * This module is the only store that does not map to a PocketBase collection,
+ * making the naming slightly misleading, even though it is a proper Svelte store.
+ * TODO: Rename to reflect its purpose more clearly.
+ */
+
+import { writable, derived, get } from "svelte/store";
+import type { Item, CustomizationValue } from "$lib/types";
+import { customizationValues } from "./menuStore";
+
+export interface CartItem extends Item {
```

Comment:
Item should ideally have the functionality of CartItem even if Item does not itself contain customizations

An `Item` in the domain is the fully expanded `Item` `OrderItem` product